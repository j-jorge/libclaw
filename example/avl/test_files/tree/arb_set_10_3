 76 22 31 78 28 17 88 62 2 57 37