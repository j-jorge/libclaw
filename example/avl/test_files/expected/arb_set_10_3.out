Tree is empty
File loading...
size : 11
Min = 2
Max = 88
Tree's not empty
Found : 2 17 22 28 31 37 57 62 76 78 88
Scan with an iterator : 2 17 22 28 31 37 57 62 76 78 88
Scan bis, iterator affectation :2: 2 17 22 28 31 37 57 62 76 78 88
17: 17 22 28 31 37 57 62 76 78 88
22: 22 28 31 37 57 62 76 78 88
28: 28 31 37 57 62 76 78 88
31: 31 37 57 62 76 78 88
37: 37 57 62 76 78 88
57: 57 62 76 78 88
62: 62 76 78 88
76: 76 78 88
78: 78 88
88: 88
Deletions
Copy and scan
source size : 0
copy size : 0
Tree's copy is empty

Reverse scan

Assignment
Clear
Tree's is empty
